server/src/main/java/org/elasticsearch/monitor/os/OsProbe.java

```diff
@@ -511,17 +511,26 @@ boolean areCgroupStatsAvailable() {
                 assert !controllerMap.isEmpty();
 
                 final String cpuAcctControlGroup = controllerMap.get("cpuacct");
-                assert cpuAcctControlGroup != null;
+                if (cpuAcctControlGroup == null) {
+                    logger.debug("no [cpuacct] data found in control group stats");
```

Comment:
I would suggest `cgroup` instead of `control group` since we refer to these as `cgroup` in JSON responses, etc.